cqf-fhir-cr/src/test/java/org/opencds/cqf/fhir/cr/measure/r4/MeasureReportTypeSummaryTest.java

```diff
@@ -0,0 +1,235 @@
+package org.opencds.cqf.fhir.cr.measure.r4;
+
+import static org.opencds.cqf.fhir.test.Resources.getResourcePath;
+
+import ca.uhn.fhir.context.FhirContext;
+import java.nio.file.Paths;
+import org.hl7.fhir.r4.model.DateTimeType;
+import org.hl7.fhir.r4.model.MeasureReport.MeasureReportStatus;
+import org.hl7.fhir.r4.model.Period;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.opencds.cqf.fhir.api.Repository;
+import org.opencds.cqf.fhir.cr.measure.r4.Measure.Given;
+import org.opencds.cqf.fhir.cr.measure.r4.utils.TestDataGenerator;
+import org.opencds.cqf.fhir.utility.repository.ig.IgRepository;
+
+/**
+ * This test is to verify and confirm that created Summary report has required format and output
+ *     subject field has reference
+ *     report has date
+ *     report type summary
+ *     has measure url
+ *     has correct status
+ *     has period
+ *     errors when missing required population
```

Comment:
There are no error tests in this test class either

Reply:
fixed description